spec/configuration_spec.rb

```diff
@@ -59,6 +59,37 @@
         end.to raise_error "Multiple entries for short_option '-f' found!".red
       end
 
+      it "raises an error for unresolved conflict between options" do
+        conflicting_options = [
+          FastlaneCore::ConfigItem.new(key: :foo,
+                                       short_option: "-f",
+                                       conflicting_options: [:bar]),
+          FastlaneCore::ConfigItem.new(key: :bar,
+                                       short_option: "-b")
+        ]
+
+        expect do
+          FastlaneCore::Configuration.create(conflicting_options, {})
+        end.to raise_error "Unresolved conflict between options: '-f' and '-b'".red
+      end
+
+      it "calls custom conflict handler when conflict happens between two options" do
+        conflicting_options = [
+            FastlaneCore::ConfigItem.new(key: :foo,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.